content/api-specs/navarino/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Navarino Release API Specification
+menuTitle: Navarino (v1.8.0)
+description: "API Specification for the Navarino release (v1.8.0) of various SODA Projects"
+weight: 5
```

Comment:
The weight decides which menu item should bubble up and which should be at the bottom in the left menu. Currently Madagascar has an incorrect weight of 5.
The weight for Madagascar release should be less than that for Lamu (4). 
The weight for Navarino release should be less than Madagascar release so that it is the first link in the API spec.

Reply:
done